scripts/load/load_questdb.sh

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env bash
+
+# Ensure loader is available
+EXE_FILE_NAME=${EXE_FILE_NAME:-$(which tsbs_load_questdb)}
+if [[ -z "$EXE_FILE_NAME" ]]; then
+    echo "tsbs_load_questdb not available. It is not specified explicitly and not found in \$PATH"
+    exit 1
+fi
+
+# Load parameters - common
+DATA_FILE_NAME=${DATA_FILE_NAME:-influx-data.gz}
+DATABASE_PORT=${DATABASE_PORT:-9000}
+DATABASE_HEALTH_PORT=${DATABASE_HEALTH_PORT:-9003}
```

Comment:
I don't find a way to find the right port using docker.
Even running:

```
docker run --network=host \
      questdb/questdb
```

Can you share  a bit more how to overcome it? I still not able to load the data.

Here is a few details of my docker output:

```

2021-05-20T13:40:32.994960Z A tcp-line-server listening on 0:9009 [fd=33]
2021-05-20T13:40:32.998807Z I server-main started
2021-05-20T13:40:32.998909Z I server-main started
2021-05-20T13:40:32.998920Z I server-main web console URL(s):

	http://172.17.0.1:9000
	http://192.168.65.4:9000
	http://192.168.65.6:9000
	http://127.0.0.1:9000

```